pkg/engine/destroy.go

```diff
@@ -91,10 +91,9 @@ func newDestroySource(
 		var err error
 		programPackages, err = gatherPackagesFromProgram(plugctx, runtime, programInfo)
 		if err != nil {
+			plugctx.Diag.Warningf(diag.Message("", "%v"), err)
```

Comment:
We probably want to prefix the message with something like "failed to gather packages from program:"
```suggestion
			plugctx.Diag.Warningf(diag.Message("", "failed to gather packages from program: %v"), err)
```

Reply:
Updated to add the prefix